blogposts/2021/release-3.34

```diff
@@ -0,0 +1,43 @@
+---
+title: "Sourcegraph 3.34 release"
+publishDate: 2021-11-24T10:00-07:00
+description: "Sourcegraph 3.34 introduces a new page with search tips for searches that return no results along with Find Implementations functionality for Go."
+tags: [blog, release]
+slug: "release/3.34"
+published: false
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.34/sourcegraph-3-34-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.34/sourcegraph-3-34-release.png
+changelogItems:
+  - description: "Added the \"no results page\", a help page providing search tips if a search returns no results."
+    url: https://github.com/sourcegraph/sourcegraph/pull/26154
+    category: Search
+  - description: "The search indexer only polls repositories that have been marked as changed. This significantly reduces load on the Git server in installations with many repositories. If you notice index staleness, you can try disabling by setting the environment variable `SRC_SEARCH_INDEXER_EFFICIENT_POLLING_DISABLED` on `sourcegraph-frontend`."
+    url: https://github.com/sourcegraph/sourcegraph/issues/27058
+    category: Search
+  - description: "Added logging of incoming Batch Changes webhooks to help site admins debug issues with webhook delivery. By default, sites without encryption will log webhooks for three days and sites with encryption will not log webhooks."
+    url: https://docs.sourcegraph.com/admin/config/batch_changes#incoming-webhooks
+    category: Admin
+  - description: "Batch Changes now requests `workflow` scope on GitHub personal access tokens to allow writing to the `.github` directory in repositories. If you have already configured a GitHub PAT for Batch Changes, we suggest adding this scope."
+    url: https://docs.sourcegraph.com/batch_changes/how-tos/configuring_credentials
+    category: Batch Changes
+  - description: "The search UI's repositories count as well as the GraphQL API's `search().repositories` and `search().repositoriesCount` have changed semantics from the set of searchable repositories to the set of repositories with matches. This results in increased search speed, and in a future release, we'll introduce separate fields for searchable repositories."
```

Comment:
```suggestion
  - description: "In order to increase search speed, the search UI's repository count and the GraphQL API's `search().repositories`/`search().repositoriesCount` have changed semantics from being the set of searchable repositories to the set of repositories with matches. In a future release, we'll introduce separate fields for searchable repositories."
```

Reply:
cc @lguychard updated this to lead with the benefit.

Reply:
Thanks, LGTM!